File: arm.64/langs/typescript/src/trip.ts
import { Triplet } from './type.js';

export function trip(): Triplet[] {
  return [
    { address: '/version', payload: '{}', options: '{"once":true}' },
    { address: '/storage', payload: '{}', options: '{"once":true}' },
    { address: 'sql.help', payload: '{}', options: '{"once":true}' },
  ];
}

File: arm.64/langs/zig/src/trip.zig
const type_mod = @import("type.zig");

pub fn trip() [3]type_mod.Triplet {
    return [_]type_mod.Triplet{
        .{ .address = "/version", .payload = "{}", .options = "{\"once\":true}" },
        .{ .address = "/storage", .payload = "{}", .options = "{\"once\":true}" },
        .{ .address = "sql.help", .payload = "{}", .options = "{\"once\":true}" },
    };
}
